test/jest/ui/formatNumber.test.ts

```diff
@@ -210,3 +213,35 @@ describe("Ram formatting", () => {
     expect(formatRam(1073741824)).toEqual("1.00EiB");
   });
 });
+describe("Filesize formatting", () => {
+  test("With default GB mode", () => {
+    // Initial settings
```

Comment:
Can you add NaN and both infs to this and the ram test? Otherwise, everything looks great!